app/routes/members-area.my-account/route.module.scss

```diff
@@ -1,4 +1,77 @@
-.underConstructionMessage {
-    text-align: center;
-    font: var(--heading5);
+.root {
+    button {
+        width: 150px;
+    }
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+}
+
+.header h2 {
+    font-size: 28px;
+}
+
+.section {
+    padding-bottom: 30px;
+    margin-bottom: 30px;
+    border-bottom: solid 1px rgba(23, 17, 13, 0.2);
+}
+
+.section h2 {
+    padding-bottom: 16px;
+}
+
+.actions {
+    display: flex;
+    gap: 12px;
+    justify-content: flex-end;
+}
+
+.form {
+    font: var(--paragraph3);
+
+    margin: 30px 0;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 24px 60px;
+
+    label {
+        display: flex;
+        flex-direction: column;
+    }
+
+    input {
+        font: var(--paragraph1);
+    }
+}
+
+.loginInfoSection {
+    margin-top: 30px;
+    display: flex;
+    align-items: start;
+    justify-content: space-between;
+
+    & > div {
+        display: flex;
+        flex-direction: column;
+    }
+
+    button {
+        width: 175px;
+    }
+}
+
+.confirmationDialog {
+    .title {
+        font-size: 32px;
```

Comment:
let's better change it inside `DialogTitle` styles .. I didn't set 32px there because we don't have such heading among defined in the typography.css.  But maybe we should